lib/facter/framework/cli/cli.rb

```diff
@@ -205,7 +208,9 @@ def add_commands_to_help
         Cli.commands
            .select { |_k, command_class| command_class.instance_of?(Thor::Command) }
            .each do |_k, command|
-          help_command_options << build_option(command['name'], [], command['description'])
+          aliases = []
+          aliases = command.options[:aliases].default unless command.options[:aliases].nil?
```

Comment:
Could not find a better way to get aliases for Thor commands without using methods_options for each command that required alias.

Reply:
This is a limitation of the subcommands in Thor, you can specify aliases for options, but not for subcommands. 
An alternative here would be to rely on the `usage` attribute, which contains all the flags that can be used to invoke a command:
```ruby
command.to_h
=> {:name=>"version", :description=>"Print the version", :long_description=>nil, :usage=>"--version, -v", :options=>{}, :ancestor_name=>nil}
```

```diff
--------------------------------------------------------------------------------------------------------------------------------------------------------------
modified: lib/facter/framework/cli/cli.rb
--------------------------------------------------------------------------------------------------------------------------------------------------------------
@@ -126,7 +126,7 @@ def arg_parser(*args)
       Facter.values(@options, args)
     end

-    desc '--version, -v', 'Print the version', hide: true
+    desc '--version, -v', 'Print the version'
     map ['--version', '-v'] => :version
     def version(*_args)
       puts Facter::VERSION
@@ -205,7 +205,11 @@ def add_commands_to_help
         Cli.commands
            .select { |_k, command_class| command_class.instance_of?(Thor::Command) }
            .each do |_k, command|
-          help_command_options << build_option(command['name'], [], command['description'])
+          help_command_options << build_option(
+            command['name'],
+            [command['usage'].split(',')[1]],
+            command['description']
+          )
         end

         help_command_options
```